circle.yml

```diff
@@ -33,13 +33,31 @@ jobs:
       - run:
           name: Clean gradle
           command: ./gradlew clean
+      - run:
+          name: Add preferences to gradle.properties file
+          command: |
+            rm -fr gradle.properties
+            echo "android.useAndroidX=true" >> gradle.properties
+            echo "android.enableJetifier=true" >> gradle.properties
+            echo "org.gradle.jvmargs=-Xmx4608M" >> gradle.properties
       - run:
           name: Download Dependencies
           command: ./gradlew androidDependencies
       - save_cache:
           paths:
             - ~/.gradle
           key: jars-{{ checksum "build.gradle" }}-{{ checksum  "MapboxAndroidDemo/build.gradle" }}
+      - run:
+          name: Create google-services.json file
+          shell: /bin/bash -euo pipefail
+          command: |
+            echo "${GOOGLE_SERVICES_JSON_RELEASE}" > MapboxAndroidDemo/google-services.json
```

Comment:
this need the same configuration as FABRIC_PROPERTIES_CONFIG_FILE below

Reply:
Adjusted